### DISCUSSION TITLE
Amateur question.  Pre-release.

### DISCUSSION THREAD

**Discussion by wc-yaa** [blitz3d-ng/package#74] [General]
Hi, I am an amateur. This language looks good! The current version is called pre-release version. The word "pre-release" means that there will be a formal version released in the future? Am I right? I am curious.

**Comment by kfprimm** [comment]
Hi,

Yes, "pre-release" is GitHub's phrasing. The way I use it is to mean that you shouldn't depend on any version tagged with this in a production context.

Those versions (currently all) have not been extensively QA'd.

Eventually, I will freeze take some time to do a thorough check of all commands and then release that as a "stable" version.

The current "edge" release isn't necessarily unstable, but there are known issues with new features (non-Windows stuff) and probably a few unknown issues with the legacy code.

Thanks.

**Reply by wc-yaa** [reply]
Hello....thanks a lot! You explained in detail！ I really hope that the stable version of Blitz3D "NG" will be popular and successful in the future. Sure. Yeah!
